arangod/RocksDBEngine/RocksDBSettingsManager.cpp

```diff
@@ -80,14 +84,11 @@ arangodb::Result writeCounterValue(
   rocksdb::Status s =
       rtrx->Put(RocksDBColumnFamily::definitions(), key.string(), value);
   if (!s.ok()) {
-    rtrx->Rollback();
-    auto rStatus = convertStatus(s);
-    LOG_TOPIC(WARN, Logger::ENGINES)
-        << "writing counters failed: " << rStatus.errorMessage();
```

Comment:
Please don't remove the log message here. If anything, replace it with one with more info, e.g.
```
LOG_TOPIC(WARN, Logger::ENGINES)
    << "writing counter for collection with objectId '" << pair.first << "' failed: " << rStatus.errorMessage();
```